src/main/java/uk/gov/ons/fwmt/job_service/exceptions/ExceptionCode.java

```diff
@@ -1,26 +1,65 @@
 package uk.gov.ons.fwmt.job_service.exceptions;
 
+import lombok.Getter;
+
 public enum ExceptionCode {
-  FWMT_JOB_SERVICE_0001("UNKNOWN"),
-  FWMT_JOB_SERVICE_0002("INVALID_FILE_NAME"),
-  FWMT_JOB_SERVICE_0003("INVALID_MEDIA_TYPE"),
-  FWMT_JOB_SERVICE_0004("UNKNOWN_JOB_ID"),
-  FWMT_JOB_SERVICE_0005("UNKNOWN_USER_ID"),
-  FWMT_JOB_SERVICE_0006("CSV_MISSING_COLUMN"),
-  FWMT_JOB_SERVICE_0007("CSV_INVALID_FIELD"),
-  FWMT_JOB_SERVICE_0008("CSV_OTHER"),
-  FWMT_JOB_SERVICE_0009("TM_UNAUTHENTICATED"),
-  FWMT_JOB_SERVICE_0010("TM_INACCESSIBLE"),
-  FWMT_JOB_SERVICE_0011("UNKNOWN_FIELD_PERIOD"),
-  FWMT_JOB_SERVICE_0012("UNABLE_TO_SAVE_FILE");
-
-  private final String error;
-  
-  ExceptionCode(String error) {
-    this.error = error;
+  // indicates an unexpected exception, usually a runtime exception
+  UNKNOWN(1, "UNKNOWN"),
```

Comment:
add the error code directly here instead of 1
like:FWMT_JOB_SERVICE_0001.
Any reason you want to keep it as number.?

Reply:
I wanted to add a human-readable name, to ensure that anyone reading the code would see something akin to `ExceptionCode.CSV_OTHER` as opposed to `ExceptionCode.FWMT_JOB_SERVICE_0008`, in order to improve clarity.

However, I wanted to ensure that when output into the logs, `FWMT_JOB_SERVICE_0008` is still printed. This is why I added the number as a parameter.

Reply:
I was mentioning this : UNKNOWN(FWMT_JOB_SERVICE_0001, "UNKNOWN")

Reply:
`FWMT_JOB_SERVICE_0001` doesn't exist, that's the symbol that I replaced.